test/test_output.jl

```diff
@@ -108,3 +108,10 @@ end
         Plots._show(io, MIME("text/html"), pl)
     end
 end
+
+@testset "size error handling" begin
+    plt = plot(size = ())
+    @test_throws ArgumentError savefig(plt, tempname())
+    plt = plot(size = ())
+    @test_throws ArgumentError savefig(plt, tempname())
```

Comment:
why is it the same test two times?

Reply:
sorry, I had fixed that before and messed up some commits